src/midi.c

```diff
@@ -666,16 +666,18 @@ static PmEvent* Midi_BuildEvent(Uint8 byte)
 	static const Uint8 shifts[] = { 0,8,16,24 };
 	static PmEvent midiEvent = { 0,0 };
 	static Uint32 midimsg;
-//	static Uint8 runningStatus = 0;
+	static Uint8 runningStatus = 0;
 	static Uint8 bytesToWait = 0;
 	static Uint8 bytesCollected = 0;
 	static bool processingSysex = false;
+	static bool expectStatus = true;
 
 	// -- status byte
-	if (byte & 0x80)
+	if ((byte & 0x80) > 0)
```

Comment:
This change does not make any sense. Or what's the difference to the previous behavior?